### DISCUSSION TITLE
I need more precision in PointCoordinate's decimal places

### DISCUSSION THREAD

**Discussion by ronnypolloqueri** [clickbar/laravel-magellan#72] [Ideas]
Please, when save a geometry in Postgres the coordinate points is rounded to only 6 decimal places (see stringifyFloat function src/Data/Geometries/GeometryHelper.php)
-18.000027344015  => -18.000027

**Comment by saibotk** [comment]
Oh sorry, that was a bug!

Thanks for reporting this, it will be fixed by #73

**Comment by saibotk** [comment]
Sorry this took so long :) but this is fixed in 1.7.1 :)
